es6-template-literals/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = `${number}${street}`;
```

Comment:
Only refactor statements with concatenation, no other line should be changed:

---

![image](https://user-images.githubusercontent.com/16143771/174712781-b866bcbf-b941-4580-8df2-f825d1419fc3.png)

---

![image](https://user-images.githubusercontent.com/16143771/174712814-0751ce8f-a605-4133-863a-c5c2793bc0af.png)